src/handlers/check_commits/non_default_branch.rs

```diff
@@ -0,0 +1,35 @@
+use crate::{config::WarnNonDefaultBranchException, github::IssuesEvent};
+
+const NON_DEFAULT_BRANCH: &str =
```

Comment:
I know that it's pre-existing, but could you please turn these strings from constants into functions (with placeholders as arguments), to make sure that we never forget to pass the required placeholders? I eventually want to do this with all such string constants in the codebase.

Reply:
Seems reasonable, done.

Also took the time to rework the exception handling since it was becoming too complex for my liking.